### ISSUE TITLE
Krames URL no longer works!

### ISSUE THREAD

**Issue by DSSUSER** [logicahealth/InfoButtons#8]
The Krames URL no longer works.

Krames apparently changed their URL.

**Comment by DSSUSER** [comment]
I am also trying to research the new URL with Krames myself, but someone took down the original XML, and I have no clue as to how to recreate a new XML file, without having the original as a template.

Please repost the original XML file, or send me a copy, so that I at least have a template to work from.

Thanks in advance,

JCF

**Comment by gdelfiol** [comment]
Krames has changed their API to institution-specific base URLs. Every institution that licenses Krames will have a custom base URL that needs to be configured in the Krames profile. Therefore, we removed the Krames profile from the resource store. The best way to configure Krames is to create a custom profile in LITE. For those who prefer to edit XML directly, we posted the XML profile to GitHub below. Krames insitution-specific base URL needs to be provided in the url attribute as in the example below:

<knowledgeRequestServiceLocation url="[add Krames base URL here]"/>

https://github.com/VHAINNOVATIONS/InfoButtons/blob/development/profilestore/Krames%20Staywell.xml
